e2e/scientists_queries.yaml

```diff
@@ -338,3 +338,33 @@ queries:
           - contains_row: ["<Albert_Einstein>", "<Nobel_Prize_in_Physics>"]
           - contains_row: ["<Albert_Fert>", "<Wolf_Prize_in_Physics>"]
           - contains_row: ["<Albert_Overhauser>", "<National_Medal_of_Science_for_Physical_Science>"]
+  - query : having-predicate-religion 
+    solutions:
+      - type: no-text
+        sparql: |
+          SELECT ?predicate (COUNT(?predicate) as ?count) WHERE {
+            ?x <is-a> <Astronaut> .
+            ?x ql:has-predicate ?predicate .
+          }
+          GROUP BY ?predicate
+          HAVING (?predicate < <Z) (?predicate = <Religion>)
```

Comment:
So these are multiple `HAVING` clauses which are treated as `AND` right? I think we should document that somewhere besides the SPARQL standard. Also I think we should have a few more examples in the e2e Tests, especially also the very basic usage. I like using them as normal examples and to look up how some syntax works and thus having a few different cases is both a useful test and an example.